docs/topics/mta-rn-resolved-issues-7-0-2.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes-7.0/master.adoc
+
+:_content-type: REFERENCE
+[id="mta-rn-resolved-issues-7-0-2_{context}"]
+= Resolved issues
+
+The following highlighted issues have been resolved in {ProductFullName} version 7.0.2.
+
+.MTA CLI does not function as expected on Arm-based machines
```

Comment:
```suggestion
```suggestion
.CVE-2022-1962: golang: go/parser stack exhaustion in all Parse* functions  Security

A flaw was found in versions of the golang standard library, go/parser before Go 1.17.12 and Go 1.18.4. When calling any Parse functions on the Go source code, which contains deeply nested types or declarations, a panic can occur due to stack exhaustion. This issue allows an attacker to impact system availability.

For more details, see link:https://access.redhat.com/security/cve/CVE-2022-1962[(CVE-2022-1962)].


.MTA CLI does not function as expected on Arm-based machines
```